deployment/README.md

```diff
@@ -1,60 +1,133 @@
    ```
-2. Run the following command to start the Key Broker Service as a Docker container
+
+   When the relying-party container is run, a startup script configures Intel KBS to use the `ADMIN_USERNAME` and `ADMIN_PASSWORD` provided in the **kbs.env** file. You'll also need to provide an attestation API key for `TRUSTAUTHORITY_API_KEY`, which you can get from the Intel Trust Authority [portal](https://portal.trustauthority.intel.com). You might also need to configure an HTTPS proxy, depending on your network configuration. In the following sample kbs.env file, you should replace the parameters marked with angle brackets, for example `<admin-username>`.
+
+   ```
+   LOG_LEVEL=INFO
+   KEY_MANAGER=kmip
+   ADMIN_USERNAME=<admin-username>  
+   ADMIN_PASSWORD=<admin-password>  
+   KMIP_VERSION=2.0
+   KMIP_SERVER_IP=127.0.0.1
+   KMIP_SERVER_PORT=5696
+   KMIP_CLIENT_KEY_PATH=/etc/pykmip/client_key.pem
+   KMIP_CLIENT_CERT_PATH=/etc/pykmip/client_certificate.pem
+   KMIP_ROOT_CERT_PATH=/etc/pykmip/ca_certificate.pem
+   PYKMIP_LOG_LEVEL=info
+   HTTPS_PROXY=<proxy url if required>
+   TRUSTAUTHORITY_BASE_URL=https://portal.trustauthority.intel.com
+   TRUSTAUTHORITY_API_URL=https://api.trustauthority.intel.com
+   TRUSTAUTHORITY_API_KEY=<API Key>
+   ```
+   > [!NOTE]
+   > The kbs.env file contains secrets, such as the Intel KBS administrator password, the KMS password or access token, and an Intel Trust Authority attestation API key. For a secure production system, you should delete the kbs.env file after initial configuration to avoid compromising the configuration secrets. However, for this demo it's not necessary to delete the kbs.env, unless you want to protect the API key.
+
+2. Run the following command to start Intel KBS as a Docker container.
    ```bash
    sudo docker run --name relying-party -d --env-file kbs.env --restart=always -p 9443:9443 relying-party:latest
    ```
 
-## TDX Demo workload
+   9443 is the port Intel KBS will listen on. The port can be changed, if necessary.
+
+## Intel TDX Demo workload
+
+This section describes how to build and deploy the demonstration workload. The workload is an application that runs a machine learning (ML) model in an Intel TDX trust domain. The ML model is assumed to contain confidential data and parameters, therefore it is encrypted when first deployed. The demo workload (attester) must obtain an Intel Trust Authority attestation token and include the token with a request to Intel KBS to retrieve the key needed to decrypt the ML model. This workflow implements a secure key release use case in passport attestation mode.
+
 ### Build
-1. Download the [Dockerfile](./sample-workload/Dockerfile) for building TDX demo workload
+1. Download the [Dockerfile](./sample-workload/Dockerfile) for building the Intel TDX demo workload.
    ```bash
    wget https://raw.githubusercontent.com/arvind5/trustauthority-samples/task/dockerfile-deployment/deployment/sample-workload/Dockerfile
```

Comment:
Did you mean to use a link to your fork, or is this temporary?

Reply:
it should be replaced with public URL before push, let me make this change now since everything else is complete.

Reply:
updated.